### ISSUE TITLE
not intuitive vcf import dialog

### ISSUE THREAD

**Issue by Danfro** [SimpleMobileTools/Simple-Contacts#1058]
1.
In this context the string ["phone storage"](https://github.com/SimpleMobileTools/Simple-Contacts/blob/master/app/src/main/res/values-de/strings.xml#L9C46-L9C46) seems not suitable. Users think it will store it on the phone. But doesn't it actually store the data in a specific app storage?

I would like to suggest those wordings that fit better in my opinion:
"App specific storage (other apps will not be able to use contacts at all"
"dedicated app storage (other apps will not be able to use contacts at all"

2.
I would think that the most common usecase is, that contacts are shared with other apps on the device - after giving permissions. When importing an vcf file, the option "phone storage... (as under 1.) is preselected. This seems not intuitive. I think it would be better if "phone" would be preselected.

**Comment by Maxitendo1** [comment]
@Danfro SimpleMobileTools apps were sold to ad and spyware company. Fork is here:

https://github.com/FossifyOrg

**Comment by Danfro** [comment]
Yes, I heard that the app was sold. But still thanks for the update. 

--
